Solutions/Rapid7InsightVM/Playbooks/Playbooks/Rapid7InsightVM-RunScan/readme.md

```diff
@@ -0,0 +1,48 @@
+# Rapid7InsightVM-RunScan
+
+## Summary
+
+When a new sentinel incident is created, this playbook gets triggered and performs the following actions:
+
+1. Obtains IPs from the incident.
+2. Searches asset ids by the IPs.
+3. Obtains a list of scan engines.
+4. Sends an adaptive card to the Teams channel where the user can choose an action to be taken.
+
+<img src="./teams_screenshot.png" width="50%"/><br>
+
+5. Runs scans for selected IPs using chosen scan engines.
+6. Add inforamtions about launched scans as a comment to the incident.
+
+<img src="./playbook_screenshot.png" width="50%"/><br>
+
+### Prerequisites
+
+1. Prior to the deployment of this playbook, Rapid7 InsightVM API Connector needs to be deployed under the same subscription.
+2. Obtain Rapid7 InsightVM API credentials. Refer to Rapid7 InsightVM API Custom Connector documentation.
+
+### Deployment instructions
+
+1. To deploy the Playbook, click the Deploy to Azure button. This will launch the ARM Template deployment wizard.
+2. Fill in the required paramteres:
+    * Playbook Name: Enter the playbook name here
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FRapid7InsightVM%2FPlaybooks%2FPlaybooks%2FRapid7InsightVM-RunScan%2Fazuredeploy.json) [![Deploy to Azure](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FRapid7InsightVM%2FPlaybooks%2FPlaybooks%2FRapid7InsightVM-RunScan%2Fazuredeploy.json)
+
+### Post-Deployment instructions
+
+#### a. Authorize connections
+
+Once deployment is complete, authorize each connection.
+
+1. Click the Azure Sentinel connection resource
```

Comment:
Name of all the Sentinel/Teams connection are very generic. Update the name to be very specific and update the playbook trigger name here.